addons/web_editor/controllers/main.py

```diff
@@ -454,14 +454,14 @@ def _load_resources(self, file_type, views, bundles_restriction, only_user_custo
         urls = []
         for bundle_data in files_data_by_bundle:
             urls += bundle_data[1]
-        custom_attachments = AssetsUtils.get_all_custom_attachments(urls)
+        custom_attachments = AssetsUtils._get_custom_attachment(urls, op='in')
```

Comment:
Should we take the opportunity to rename it `_get_custom_attachments()` as this can actually return multiple attachments?

Reply:
Personally, I would name it singular as the default `op` is meant to only return one. And in this case, singular or not, I would not rename it for a dubious reason :wink: Probably the point why I made a duplicate `get_all_custom_attachments` initially but now that I reconsidered it, it seemed dumb.

Reply:
Fair deal, can't agree on everything :D